python/private/hermetic_runtime_repo_setup.bzl

```diff
@@ -67,19 +69,23 @@ def define_hermetic_runtime_toolchain_impl(
             exclude = [
                 # Unused shared libraries. `python` executable and the `:libpython` target
                 # depend on `libpython{python_version}.so.1.0`.
-                "lib/libpython{major}.{minor}.so".format(**version_dict),
+                "lib/libpython{major}.{minor}*.so".format(**version_dict),
```

Comment:
Just checking/curious: I'm guessing the `libX.Y*` pattern works because the extracted standalone interpreters only contain appropriately relevant .so files?

Reply:
This is for exclusion, that is why `libX.Y*.so` is just as good as `libX.Y.so`.